ledger/participant-integration-api/src/test/suite/scala/platform/store/dao/JdbcLedgerDaoH2DatabaseAppendOnlySpec.scala

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.platform.store.dao
+
+import org.scalatest.flatspec.AsyncFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+// Aggregate all specs in a single run to not start a new database fixture for each one
+final class JdbcLedgerDaoH2DatabaseAppendOnlySpec
```

Comment:
@rautenrieth-da can you please verify that this is the only suite which is needed for append-only in participant-integration-api?